CTP/README.md

```diff
@@ -122,40 +124,65 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
   
 - **SHELL**
   - Prepare
-	* Prepare at least two accounts (e.g., one account ``controller`` as controller, another account ``shell_instance1`` as test instance)
-	* Check out CTP for each account and configure environment variables for CTP ``controller`` and test instance ``shell_instance1``
-	
-	  ```
-	   // on controller node
+        * For local instance test
+	   * Prepare one account to use for controller and test instance
+	   * CTP and test scenario should be checked out on it, and configure JAVA_HOME
+	   ```
+	   JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
+	   ```
+	   * **Example** ``shell.conf`` for local instance, scenario and test build:
+	   ```
+	   # Configure parameters for cubrid.conf following the format outlined below, ie, default.cubrid.cubrid_port_id=1523
+	   default.cubrid.<property>=<property value> 
+	   # Configure parameters for cubrid_broker.conf following the format outlined below
+	   default.broker1.<property>=<property value> 

```

Comment:
It's better to point out that broker1 represent '%query_editor' and broker2 represent '%BROKER1' in cubrid_broker.conf